integration-test/src/test/java/org/sagebionetworks/ITTwoFactorAuthTest.java

```diff
@@ -452,6 +458,44 @@ public void testReset2FaWithTokenWithPassword(SynapseAdminClient adminClient) th
 		}
 	}
 	
+	@Test
+	public void testEnable2FaEnableRequirement(SynapseAdminClient adminClient, SynapseClient synapseClient) throws SynapseException {
```

Comment:
when ITTestExtension sets up the Admin, enable 2fa for the admin.  This would allow any test to test MFA features using that admin.  This test could validate 2fa required with the other user (admin toggles the feature).